ansible/library/counter_facts.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+from ansible.module_utils.basic import *
+import re
+import json
+from pprint import pprint
+
+
+DOCUMENTATION = '''
+---
+module: counter_facts
+version_added: "1.0"
+short_description: Retrive DUT counters.
+description:
+    - Retrieve DUT counters by parsing specific CLI commands output. Module can be extendent
+      to support more counters.
+options:
+    flex_counters:
+      description:
+        - Execute 'counterpoll show' CLI command on the DUT. Parse and return output
+          in dictionary format.
+          Will be executed only when specified, no value required.
+      default: null
+    interfaces_counters: 
```

Comment:
Can flex_counters and interfaces_counters be used together?

Reply:
Yes. Just pass them both to the 'counter_facts' module as parameters.